test/onnx/test_pytorch_jit_onnx.py

```diff
@@ -0,0 +1,56 @@
+# Owner(s): ["module: onnx"]
+import torch
+import onnxruntime
+import unittest
+from test_pytorch_onnx_onnxruntime import run_ort, ort_compare_with_pytorch
+from torch._C import parse_ir
+
+
+class _TestJITIRToONNX:
+    """Abstract base class for test cases.
+
+    Intentionally not a sub-class of unittest.TestCase so that unittest / pytest
+    don't run it directly. unitest.TestCase is mixed in as another base class when
+    creating concrete sub-types. See MakeTestCase().
+    """
+    opset_version = -1  # Sub-classes must override
+    ort_providers = ["CPUExecutionProvider"]
+
+    def run_test(self, graph_ir, example_inputs):
+        graph = parse_ir(graph_ir)
+        jit_outs = torch._C._jit_interpret_graph(graph, example_inputs)
+
+        f = torch.onnx.utils._export_jit_graph_to_onnx_model_proto(
+            graph,
+            torch.onnx.OperatorExportTypes.ONNX)
+        ort_sess = onnxruntime.InferenceSession(f, providers=self.ort_providers)
+        ort_outs = run_ort(ort_sess, example_inputs)
+
+        ort_compare_with_pytorch(ort_outs, jit_outs, rtol=1e-3, atol=1e-7)
+
+    def test_example_ir(self):
+        graph_ir = """
+        graph(%1 : Float(2, 3),
```

Comment:
As there any way we could test with an IR graph that is being extracted from a dummy model?

Reply:
Not sure I understand the question. The goal of this test suite is to enable testing conversion from IR graph to onnx. Other test suites are more suited for testing models.